xml/security_ldap.xml

```diff
@@ -389,290 +387,57 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
-      </para>
-     </step>
-     <step>
-      <para>
-       Check the <guimenu>Details</guimenu> tab for the group membership,
-       login shell, and home directory of the new user. If necessary, change
-       the default to values that better suit your needs.
-<!-- The default values
-       (as well as those of the password settings) can be defined with the
-       procedure described in
-       <xref linkend="sec.security.auth.yast.client.conf.mod"/>.-->
-      </para>
-     </step>
-     <step>
-      <para>
-       Modify or accept the default <guimenu>Password Settings</guimenu>.
-      </para>
-     </step>
-     <step>
-      <para>
-       Enter the <guimenu>Plug-Ins</guimenu> tab, select the LDAP plug-in,
-       and click <guimenu>Launch</guimenu> to configure additional LDAP
-       attributes assigned to the new user.
-<!-- (see
-       <xref linkend="fig.ldap.y2.adset"/>).-->
-      </para>
-     </step>
-    </substeps>
-   </step>
-   <step>
-    <para>
-     Click <guimenu>OK</guimenu> to apply your settings and leave the user
-     configuration.
-    </para>
-   </step>
-  </procedure>
-
-<!--
-  <figure id="fig.ldap.y2.adset">
-   <title>&yast;: Additional LDAP Settings</title>
-   <mediaobject>
-    <imageobject role="fo">
-     <imagedata fileref="ldap_y2_adset.png" width="75%" format="PNG"/>
-    </imageobject>
-    <imageobject role="html">
-     <imagedata fileref="ldap_y2_adset.png" width="75%" format="PNG"/>
-    </imageobject>
-   </mediaobject>
-  </figure>
-  -->
-
-  <para>
-   The initial input form of user administration offers <guimenu>LDAP
-   Options</guimenu>. This allows you to apply LDAP search filters to the
-   set of available users. Alternatively open the module for configuring
-   LDAP users and groups by selecting <guimenu>LDAP User and Group
-   Configuration</guimenu>.
-  </para>
- </sect1>
-
- <sect1 xml:id="sec.security.ldap.slapd" os="sles;osuse">
-  <title>Manually Configuring an LDAP Server</title>
-
-  <para>
-   <remark>taroth 2019-02-21: @wbrown: the content of this section needs to be
-   revised for sure - could you please let me know what to remove, replace, or
-   add?</remark>
-   To conveniently access the configuration back-end, you use SASL external
-   authentication. For example, the following <command>ldapsearch</command>
-   command executed as &rootuser; can show the complete
-   <command>slapd</command> configuration:
-  </para>
-
-<screen>&prompt.user;ldapsearch -Y external -H ldapi:/// -b cn=config</screen>
+    </varlistentry>
+  </variablelist>
 
+  <!--taroth 2019-02-25: the following note and para are 'leftovers' of the former
+   section, check with wbrown if we need to keep them-->
   <note>
-<!-- bnc#911409 -->
    <title>LDAP Server Is Part of the Authentication Server</title>
    <para>
+    <remark>taroth 2019-02-25: @wbrown: is this note still valid?</remark>
     Basic LDAP Server initialization and configuration can be done within
```

Comment:
I'm not sure. I would assume that yast if it can configure a 389-ds instance, would make this comment valid?

Reply:
That's exactly the question I had. c#13 in Fate#323362 mentions an integration with YaST and according to one of the project managers, yast-auth-server was updated in SLE 15 to handle 389-ds server. But I don't know who integrated this and if it works as expected/described in the docs. I assumed you would know about the YaST part of this feature as well, but I will check with Frederic then.

Reply:
Yeah, I don't know about the yast side at all. For SUSE, my employment is for upstream focus, and packaging. I haven't used yast. I'd love to be in contact with the person who did the yast integration work though as I think 389-ds is changing a lot and so there may be some "rework" to do.

Reply:
Will send details via mail.